DOCUMENTATION.md

```diff
@@ -325,145 +322,143 @@ The `package` property is an array list of packages to be outputted. If the prop
+| deb          | &#10004; |        -         |              -              |
+| rpm          | &#10004; |        -         |              -              |
+| freebsd      | &#10004; |        -         |              -              |
+| pacman       | &#10004; |        -         |              -              |
+| p5p          | &#10004; |        -         |              -              |
+| apk          | &#10004; |        -         |              -              |
+
+- **[1]** Only Window 10 can build AppX packages.
+
+#### Setting the Package `arch`
 
 The `arch` property is an array list of architectures that each package is built for. When the property is defined, the default arch is not used unless added.
 
-> :warning:  Not all architectures are available for every operating system. Please review the [Electron Releases](https://github.com/electron/electron/releases/) to identify valid combinations. For example, macOS (Darwin) only supports x64.
+> &#10071;  Not all architectures are available for every operating system. Please review the [Electron Releases](https://github.com/electron/electron/releases/) to identify valid combinations. For example, macOS (Darwin) only supports x64.
 
 **Available Arch**
-* ia32
-* x64
-* armv71
-* arm64
 
+- ia32
+- x64
+- armv71
+- arm64
 
 The example above will generate an `x64` `dmg` package.
 
-```
+```json
 {
-  "electron": {
-    "mac": {
-      "package": [
-        "dmg"
-      ],
-      "arch": ["x64"]
+    "electron": {
+        "mac": {
+            "package": [ "dmg" ],
+            "arch": [ "x64" ]
+        }
     }
-  }
 }
 ```
 
 ### Multi-Platform Build Support
 
-> :warning: Not all platforms support this feature and may have limitations.
+> &#10071; Not all platforms support this feature and may have limitations.
 
 Building for multiple platforms on a single operating system may possible but has limitations. It is recommended that the builder's operating system (host OS) matches with the platform that is being built.
 
 The matrix below shows each host OS and for which platforms they are capable of building applications.
 
-| Host <sup>**[1]**</sup> | Linux              | Mac                | Window                       |
-| ----------------------- | :----------------: | :----------------: | :--------------------------: |
-| Linux                   | :white_check_mark: |                    | :warning: <sup>**[2]**</sup> |
-| Mac <sup>**[3]**</sup>  | :white_check_mark: | :white_check_mark: | :warning: <sup>**[2]**</sup> |
-| Window                  |                    |                    | :white_check_mark:           |
+| Host <sup>**[1]**</sup> |       Linux        |        Mac         |           Window            |
+| ----------------------- | :----------------: | :----------------: | :-------------------------: |
+| Linux                   | :white_check_mark: |                    | &#10071; <sup>**[2]**</sup> |
```

Comment:
`:white_check_mark:` could be changed to `&#10004;`or `&#9989;` to keep consistency. I would prefer the second one since it's a bit more distinctive and eye-catching.